### PR TITLE
fix: enabling lograge in superadmin path

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

- Fixes https://github.com/chatwoot/chatwoot/issues/6922
- Fixes https://linear.app/chatwoot/issue/CW-1549/bug-superadmin-page-fails-to-load-when-lograge-is-enabled
- Add spec for super admin devise session controller

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Tested locally

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by sojan-official — COMMENTED** [review]
Can we also add a spec for the super admin login page with climate control setting log_rage  env variable to true. ensuring the page is not breaking

**Comment by vishnu-narayanan** [comment]
> Can we also add a spec for the super admin login page with climate control setting log_rage env variable to true. ensuring the page is not breaking

@sojan-official done

File: spec/controllers/super_admin/devise/sessions_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'Super Admin', type: :request do
+  describe '/super_admin' do
+    it 'renders the login page' do
+      with_modified_env LOGRAGE_ENABLED: 'true' do
+        get '/super_admin/sign_in'
+        expect(response).to have_http_status(:ok)
+        expect(response).to render_template(:new)
```
Comment (sojan-official) [inline spec/controllers/super_admin/devise/sessions_controller_spec.rb:9]:
```suggestion
        expect(response).to render_template(:new)
```
wouldn't require this line i guess

**Review by sojan-official — APPROVED** [review]
LGTM

**Pull request merged**